versions/3.1.0.md

```diff
@@ -2241,10 +2243,18 @@ description: Pets operations
 
 A simple object to allow referencing other components in the OpenAPI document, internally and externally.
 
+The `$ref` string value contains a URI [RFC3986](https://tools.ietf.org/html/rfc3986), which identifies the location of the value being referenced.
+
+Resolution of a Reference Object SHOULD yield the referenced value. Implementations MAY choose to replace the reference with the referenced value.
+
+If the URI contained in the `$ref` value is a relative URI, then the Base URI resolution MUST be calculated according to [RFC3986](https://tools.ietf.org/html/rfc3986#section-5.2). Resolution is performed relative to the referring document.
```

Comment:
As far as a I understand it, there is no such thing as a "relative URI".

Reply:
s/relative URI/a URI that is not absolute/ (i.e. a uri-reference)